tools/autograph-client/client.go

```diff
@@ -508,6 +515,9 @@ func verifyMAR(signedMAR []byte) bool {
 }
 
 func verifyPGP(input []byte, signature string, pubkey string) bool {
+	return true
```

Comment:
nit: we might want to log that these are no-ops

Reply:
👍